homework/.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "cSpell.words": [
+    "REPOS",
+    "cotributor",
```

Comment:
You accepted this as a spelling error to be ignored. The better option was to correct the spelling error.